HHS/Head-Start-TTADP
# [TTAHUB-966] user email verification flow

## Description of change

Depends on TTAHUB-758. Will stay in draft status until #1004 is merged into the integration branch. At that point, the diff for this PR should be much easier to review.

## How to test

There's a new dependency (`jsonwebtoken`) so you'll need to `yarn install`.

- Visit the account management page.
- Notice the message that indicates your email address has not been verified.
- Click the button to send off a verification email.
- Check mailcatcher to ensure the mail was received and click on the included link.
- Ensure you are redirected to the account management page and a success alert shows your email has been verified and you can now customize your email preferences.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-966


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
